docs/RFCS/range_repair.md

```diff
@@ -0,0 +1,94 @@
+- Feature Name: Range Repair
```

Comment:
Hmmm, it's not really repairing a range. Perhaps we should call this "Range replica reincarnation"